### ISSUE TITLE
Bring back Wallet History page (transaction list data)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#4052]
We should still have a way to access transaction list data since it's more detailed than the current list and shows fees/abandons etc. It also allows export to xls and some creators use this for tracking their income. 

It needs to be properly paginated, but no filters needed.

**Comment by bitcoinbrisbane** [comment]
Wouldn't `.csv` be better than `.xls`?

**Comment by kauffj** [comment]
Yes, `.csv` over `.xls` 100%. It's probably pretty workable to add an export button to the current state of Transaction List. I added help wanted to this issue.

**Comment by jeffslofish** [comment]
I'm working on this now.

**Comment by jeffslofish** [comment]
What if we have a button on the wallet page called "Detailed History" that takes you to the old transactions page which has all the transaction list data. Or did you want to display the transaction list data right on the wallet page? I have the former working right now ... it would take some more work to display it on the wallet page.


![Screen Shot 2020-05-16 at 12 40 59 AM](https://user-images.githubusercontent.com/1240484/82113923-491a0600-970e-11ea-957a-27660b270fda.png)

**Comment by kauffj** [comment]
Can we instead do this in a way that you "toggle" between extra details mode? 

I dislike two distinct pagination methods that are 80% similar pretty strongly from a UX perspective.

**Comment by jeffslofish** [comment]
Is this more along the lines you were thinking?

https://open.lbry.com/@jeffslofish:7/wallet-history-test-2:3

**Comment by kauffj** [comment]
@jeffslofish yes. Additional comments:

- Would like a shorter label. If all that is added is fees, it can be labeled "Show Fees"
- Ideally height would not "jump" when toggling. Can we show a busy indicator without trashing the current results?
- The table looks like it could be smarter about widths (existing issue, but while you're there... :grin:)
- Can the flow to export be presented as one simple button labeled "Export"? The preparation can begin when the user requests it, rather than presenting as two steps

**Comment by kauffj** [comment]
@jeffslofish also, if you ever want prompt UI feedback, do not hesitate to @ me in Slack. I will almost always see and answer GitHub messages too, but I process these as low priority emails.
